packages/main/src/components/channel/channeldetails/ChannelDetails.jsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styles from "./channeldetail.module.css";
+import cancel from "../../../assets/images/close.png";
+import star from "../../../assets/images/star.png";
+import calling from "../../../assets/images/call-calling.png";
+import notify from "../../../assets/images/notification.png";
+import copy from "../../../assets/images/copy.png";
```

Comment:
These are things you can get from react icons and style
you are bloating the app